tango_simlib/tests/test_tango_sim_generator.py

```diff
@@ -95,6 +96,18 @@ def test_device_command_list(self):
                           "The commands specified in the xmi file are not present in"
                           " the device")
 
+    def test_device_property_list(self):
+        """Testing whether the device properties in the model are added to
+        the TANGO sim device as expected
+        """
+        db = PyTango.Database()
```

Comment:
I think you should mock this database object.  You don't want the tests to try to connect to a real database.  Also when you start up this test device, it is running in the nodb mode.  Best we can do is check that `put_device_property` was called on the mock with correct parameters.